.github/workflows/build_and_push.yaml

```diff
@@ -0,0 +1,52 @@
+# A workflow that builds and (optionally) pushes the Docker container image
+# artifacts for the application. The build action occurs on pull request events
+# that target the `main` branch, and the push action occurs only with tagged releases
+# and ticket branches.
+---
+name: "Build and Push"
+
+"on":
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'src/lsst/cmservice/**'
+  push:
+    tags:
+      - "*"
+  workflow_dispatch:
+
+jobs:
+  ci:
+    uses:
+      ./.github/workflows/ci.yaml
+
+  build:
+    name: "Build and Push Application Container Images"
+    needs:
+      - ci
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
```

Comment:
Is there any reasonable situation in which 20 minutes wouldn't be enough? Probably not but worth thinking about for a minute if you didn't already.

Reply:
I think this is an inherited value from ultimately a square template, but it is a reasonable value if a bit long. 20 minutes would be an eternity for a build.